src/lib/validations/sql.js

```diff
@@ -156,7 +156,8 @@ const checks: Checks = {
 		recommendation: 'Remove these lines',
 	},
 	trigger: {
-		matcher: /TRIGGER/,
+		// Match `CREATE (DEFINER=`root`@`host`) TRIGGER`
+		matcher: /^CREATE (DEFINER=`?(\w*)(`@`)?(\w*\.*%?)*`?)?(| )TRIGGER/,
```

Comment:
Now that it's more defined, we can probably make this case-insensitive.  Especially if we ship #681 alongside.

Reply:
This doesn't seem to match the string:

```
CREATE (DEFINER=`root`@`host.fdsa.com`) TRIGGER
```

...unless I'm missing something.
https://regex101.com/r/9pCSWm/1

Reply:
Oops, I was missing the optional parentheses around the definer. Fixed the regex to be case-insensitive + match the example string 🙇‍♀️